nixos/modules/services/search/elasticsearch.nix

```diff
@@ -147,9 +145,7 @@ in
       path = [ pkgs.inetutils ];
       environment = {
         ES_HOME = cfg.dataDir;
-        ES_JAVA_OPTS = toString (optional (!es6) [ "-Des.path.conf=${configDir}" ]
-          ++ cfg.extraJavaOptions);
-      } // optionalAttrs es6 {
+        ES_JAVA_OPTS = toString cfg.extraJavaOptions;
```

Comment:
The formatting done in the previous commit is especially not needed if this part gets removed eventually.